testsuite/cli/src/dev_commands.rs

```diff
@@ -175,6 +176,33 @@ impl Command for DevCommandDisableCustomScript {
     }
 }
 
+pub struct DevCommandChangeLibraVersion {}
+
+impl Command for DevCommandChangeLibraVersion {
+    fn get_aliases(&self) -> Vec<&'static str> {
+        vec!["change_libra_version"]
+    }
+
+    fn get_params_help(&self) -> &'static str {
+        ""
```

Comment:
may put a params message here, by reading the diff, I'm not even sure what is 2 params, one is version seems clear, what is next one? I understand it is inside change_libra_version func, but, some help message will help a lot for user.